TodoApp/src/TodoApp.Api/Controllers/ItemsController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Net;
+using System.Threading.Tasks;
+using System.Web.Http;
+using TodoApp.Contracts;
+using TodoApp.Contracts.Helpers;
+using TodoApp.Contracts.Models;
+
+namespace TodoApp.Api.Controllers
+{
+    public class ItemsController : ApiController
+    {
+        private readonly Guid _guidOfPostItem = new Guid("e6eb4638-38a4-49ac-8aaf-878684397707");
```

Comment:
Just a food for thought: 
You only use this attribute in the `POST` method. However, the repository returns the posted item and you can just take the id from it. It's fine for this point in time, as the controller is supposed to be dummy, you will definitely have to change it in the future though.